packages/dev/core/src/Physics/v2/index.ts

```diff
@@ -5,3 +5,4 @@ export * from "./physicsShape";
 export * from "./physicsConstraint";
 export * from "./physicsMaterial";
 export * from "./physicsAggregate";
+export * from "./Plugins/index";

```

Comment:
Add this when you are exporting something out of this file, otherwise the build will fail

Reply:
That's what I just realized :)